### PR TITLE
Make access control cluster attributes nullable

### PR DESCRIPTION
**Pull request by mlepage-google**
Couldn't do this before as the tooling and/or generated code was not up to it.
Now it seems to be able to handle the list attributes being nullable.

#### Problem
Access Control cluster attributes should be nullable. This couldn't be done when they were created
as the tooling and generated code were not up to it.

#### Change overview
Now that this is feasible, make the attributes in access-control-cluster.xml nullable.

#### Testing
Built and ran all-clusters-app on Linux with REPL to try null lists
as well as lists with items.

### CODE REVIEW

**Comment by mlepage-google** [comment]
Ideally related PR #13607 is merged first, though I think this will be OK if it does go in beforehand, it just won't actually handle null as an attribute value until it does, it'll just complain with an error if you try.

**Review by bzbarsky-apple — COMMENTED** [review]
I don't quite follow this change.  These are not marked nullable in the spec...

File: src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml [hunk 1]
```diff
@@ -67,13 +67,13 @@ limitations under the License.
       and enforce Access Control for the Node’s endpoints and their associated
       cluster instances.</description>
     <!-- Base data types -->
-    <attribute side="server" code="0x0000" define="ACL" type="ARRAY" entryType="AccessControlEntry" writable="true">
+    <attribute side="server" code="0x0000" define="ACL" type="ARRAY" entryType="AccessControlEntry" isNullable="true" writable="true">
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml:70]: This is not nullable in the spec.  Why are we marking it nullable?

**Comment by mlepage-google** [comment]
The attributes were originally nullable in the spec, but I see they lost it at some point before we settled on final spec. I'll withdraw this PR.

**Pull request closed without merging**